tools/thumbnaild-video/libavthumbnailer.cpp

```diff
@@ -204,6 +204,8 @@ QImage createVideoThumbnail(const QString &fileName, const QSize &requestedSize,
             } else if (ret != AVERROR(EAGAIN)) {
                 av_packet_unref(&packet);
                 return image;
+            } else {
+                break;
```

Comment:
Should there be some warning or error handling from the codec?

The fix works good on Xperia 1.

Reply:
Added a comment to clarify the code.